gui/saveload.cpp

```diff
@@ -77,14 +77,24 @@ Common::String SaveLoadChooser::createDefaultSaveDescription(const int slot) con
 
 int SaveLoadChooser::runModalWithCurrentTarget() {
 	const Plugin *plugin = EngineMan.findPlugin(ConfMan.get("engineid"));
```

Comment:
You could avoid many renames if you would keep `plugin` variable and introduce `metaEnginePlugin` or something. This is for all these files in the GUI.